requirements-dev.txt

```diff
@@ -8,7 +8,7 @@ numpydoc
 pydocstyle
 pylint
 pyro-ppl
-tensorflow
+tensorflow==1.31.1
```

Comment:
```suggestion
tensorflow==1.13.1
```

Reply:
good catch

https://pypi.org/project/tensorflow/#history

It's an image there, so I couldn't copy-paste